src/hotspot/share/cds/cdsConfig.hpp

```diff
@@ -135,7 +169,6 @@ class CDSConfig : public AllStatic {
   static void stop_dumping_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
   static void stop_using_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
 
-
```

Comment:
Blank line revmoved by accident?

Reply:
There were two empty lines, so I removed one of them.